integrated_channels/blackboard/models.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+Database models for Enterprise Integrated Channel Blackboard.
+"""
+
+from simple_history.models import HistoricalRecords
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from integrated_channels.integrated_channel.models import EnterpriseCustomerPluginConfiguration
+from .transmitters.content_metadata import BlackboardContentMetadataTransmitter
+from .exporters.content_metadata import BlackboardContentMetadataExporter
+
+
+@python_2_unicode_compatible
+class BlackboardEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguration):
+    """
+    The Enterprise-specific configuration we need for integrating with Blackboard.
+
+    .. no_pii:
```

Comment:
What is this line doing?

Reply:
I believe it is related to this 
https://github.com/edx/edx-enterprise/blob/4a598e8b6a908f474b533a61cbba2ea2667e6d48/.annotation_safe_list.yml#L1

that file handles autogenerated models, and this here is for manually annotating them for whatever task is checking these ... I need to learn more but pretty sure we needed to add these ...

Reply:
Oh, interesting. :)